test/app/views/play/level/tome/Problem.spec.coffee

```diff
@@ -28,29 +28,27 @@ describe 'Problem', ->
     type: 'runtime'
   }
   levelID = 'awesome'
-  
+
   describe '.translate()', ->
-    beforeEach ->
-      @oldLang = $.i18n.lng()
-      $.i18n.setLng('rot13')
-    afterEach ->
-      $.i18n.setLng(@oldLang)
     it 'translates messages with line numbers, error types, and placeholders', ->
-      english = 'Line 12: ReferenceError: somethin is not defined'
-      rot13 = 'Yvar 12: ErsreraprReebe: somethin vf abg qrsvarq'
-      expect(Problem.prototype.translate(english)).toEqual(rot13)
-      english = "`foo`'s argument `bar` has a problem. Is there an enemy within your line-of-sight yet?"
-      rot13 = "`foo`'f nethzrag `bar` unf n ceboyrz. Vf gurer na rarzl jvguva lbhe yvar-bs-fvtug lrg?"
-      expect(Problem.prototype.translate(english)).toEqual(rot13)
-      english="""
-        `attack`'s argument `target` should have type `unit`, but got `function`.
-        Target a unit.
-      """
-      rot13="""
-        `attack`'f nethzrag `target` fubhyq unir glcr `unit`, ohg tbg `function`.
-        Gnetrg n havg.
-      """
-      expect(Problem.prototype.translate(english)).toEqual(rot13)
+      oldLang = $.i18n.language
+      $.i18n.changeLanguage 'rot13', ->
+        english = 'Line 12: ReferenceError: somethin is not defined'
+        rot13 = 'Yvar 12: ErsreraprReebe: somethin vf abg qrsvarq'
+        expect(Problem.prototype.translate(english)).toEqual(rot13)
+        english = "`foo`'s argument `bar` has a problem. Is there an enemy within your line-of-sight yet?"
+        rot13 = "`foo`'f nethzrag `bar` unf n ceboyrz. Vf gurer na rarzl jvguva lbhe yvar-bs-fvtug lrg?"
+        expect(Problem.prototype.translate(english)).toEqual(rot13)
+        english="""
+          `attack`'s argument `target` should have type `unit`, but got `function`.
+          Target a unit.
+        """
+        rot13="""
+          `attack`'f nethzrag `target` fubhyq unir glcr `unit`, ohg tbg `function`.
+          Gnetrg n havg.
+        """
+        expect(Problem.prototype.translate(english)).toEqual(rot13)
+        $.i18n.changeLanguage oldLang
```

Comment:
Do other tests rely on this apparent "clean up step" for this test? If not, why do we need to set the language back at the end of the test? 

Asking because of the asynchronous nature of `changeLanguage()` and the removal of the `beforeEach` and `afterEach`, as this could trip up future tests we write for this when none of this uses `co.wrap` or `yield`'s.

Reply:
They don't appear to fail because of it. Even though the API changed to async, because we aren't loading from the network here, it's immediate. Maybe not a problem.